src/Pages/gyuyoung/Login/Login.js

```diff
@@ -1,181 +1,93 @@
   }
+  else{this.setState({type : 'password', show : 'Show'})}
+}
 
-  changePwType=()=>{
-      if(this.state.show === 'Show'){
-          this.setState({show : 'Hide', type : 'text'});
-      }
-      else{this.setState({show : 'Show', type : 'password'});}
-    }
-
-render () { 
-
-    /* 
-    CSS 적용법 조사완료 8/7일
-
-    문제는 스테이트에서 value를 챙겨와서 바꾸고 싶은데...
-    어떻게 써도 안되네...
-
-    ```
-    state={
-        Btn : 'background : blue'
-    }
-    const color={
-        this.state.Btn
-    }
-    ```
-    좀더 현명한 사고방식을 가지자
-    항상 이걸 왜 이렇게 코딩했는지 스스로에게 물어봐라
-    
-    이렇게 CSS있는지 처음알았음 8/8일
-     */
-    const color ={
-        background : this.state.Btn,
-    }
-    const fontId ={
-        fontsize : this.state.fontId,
-        transform : this.state.transformId,
-    }
-    const fontPw ={
-        fontsize : this.state.fontPw,
-        transform : this.state.transformPw,
-    }
+render () {
     return <div class="loginContainer">
     {/* 로그인화면 배경 및 테두리 */}
-    <div class="loginBackground">
-        {/* 로고 */}
-        <img alt="instagramTextLogo" src="images/gy/logo_text.png" />
-        {/* 아이디와 패스워드 입력 칸 */}
-        <div class="loginInteractivePart">
-            <div class="inputPart">
-                <div class="inputBoard">
-                    <div class="textInInputBox" style={fontId}>Phone number, username, or email</div>
-                    <input type="text" value={this.state.Id} onChange={this.handleChange1}/>
-                </div>
-                <div class="inputBoard">
-                    <div class="textInInputBox" style={fontPw}>Password</div>
-                    <input type={this.state.type} value={this.state.Pw} onChange={this.handleChange2} />
-                    <button onClick={this.changePwType} class="showBtn">{this.state.show}</button>
-                </div>
-            </div>
-        <button class="loginButton" style={color}>Log In</button>
-        <div class="otherLogin">
-            <div class="OR">
-                <div></div>
-                <span class="test">OR</span>
-                <div></div>
-            </div>
-                <button onclick="location.href='www.facebook.com/login/device-based/regular/login'" class="facebookLoginBtn">
-                    <img alt="facebookLogo" src="images/gy/facebook_logo.jpg"/>
-                    <span>Log in with Facebook</span>
-                </button>
-                <a class="forgotPw" herf="https://www.instagram.com/accounts/password/reset/">Forgot password?</a>
-            </div>
+  <div class="loginBackground">
+    {/* 로고 */}
+    <img alt="instagramTextLogo" src="images/gy/logo_text.png" />
+    {/* 아이디와 패스워드 입력 칸 */}
+    <div class="loginInteractivePart">
+      <div class="inputPart">
+        <div class="inputBoard">
+          <div class={this.state.idInInputBoxPlaceholderChanged}>Phone number, username, or email</div>
+          <input type="text" value={this.state.Id} name="id" onChange={this.InPutChangeHandler}/>
```

Comment:
```suggestion
          <input type="text" value={this.state.Id} name="id" onChange={this.inputChangeHandler}/>
```
camel case 표기법을 잘 지켜주세요